SCSS/Imports/_museum.scss

```diff
@@ -0,0 +1,46 @@
+h2 {
+  font-size: 22px;
+}
+
+body,
+.btn:hover,
+.btn:focus{
+  background-color: $darkGreen;
+}
+
+.card {
+  background-color: $limeGreen;
+  color: $darkGreen;
+}
+
+.btn {
+  background-color: $red;
+  color: #ffffff;
+}
+
+.btn:hover{
+  color: #ffffff;
+}
+.btn:focus{
+  color: #ffffff;
+}
+
+.card-body {
+  height: 80px;
+  justify-content: space-between;
+}
+
+.row {
+  margin-right: 0;
+}
+
+.dino-img-container {
+  position: relative;
+  width: 250px;
+  height: 250px;
+  overflow: hidden;
+  border: 1px solid $darkGreen;
+  background-repeat: no-repeat;
+  background-position: center top;
+  background-size: cover;
```

Comment:
Can you give this a max-width of 100% as well?

Unlikely to ever need it, but on a realllllllly tiny screen size, the images start spilling out of the card